### PR TITLE
Invalid bookings cancellation

### PR DESCRIPTION
**Pull request by mingazhev**
1. "All unpaids bookings should be cancelled - with no dependency on deadline or payment method" - this logic was implemented
2. Tests

### CODE REVIEW

File: Api/Controllers/InternalBookingsController.cs [hunk 1]
```diff
@@ -25,16 +25,15 @@ public class InternalBookingsController : BaseController
 
 
         /// <summary>
-        ///     Gets bookings for cancellation by deadline date
+        ///     Gets bookings for cancellation
         /// </summary>
-        /// <param name="deadlineDate">Deadline date</param>
         /// <returns>List of booking ids for cancellation</returns>
-        [HttpGet("cancel/{deadlineDate}")]
+        [HttpGet("cancel")]
```
Comment (hasmikharutyunyan) [inline Api/Controllers/InternalBookingsController.cs:31]: maybe `for-cancel` ?

Comment (mingazhev) [reply Api/Controllers/InternalBookingsController.cs:31]: maybe get-for-cancel? to be a verb

Comment (hasmikharutyunyan) [reply Api/Controllers/InternalBookingsController.cs:31]: I think method already marked as `httpget` and no need of `get` in route.

Comment (mingazhev) [reply Api/Controllers/InternalBookingsController.cs:31]: I'll rethink that in the next iteration

**Pull request merged**